### ISSUE TITLE
Duplication of fragments arguments during onSaveInstance

### ISSUE THREAD

**Issue by AndroFARsh** [airbnb/mavericks#545]
If fragment has more than one viewModel, arguments would be duplicated for each on save instance state.
Also we use androidx.save lib which also save arguments.

And it leads to TTL exception if args has big size

**Comment by gpeal** [comment]
@AndroFARsh Do you have very large args or lots of ViewModel? It's generally recommended to have small args and few view models

**Comment by AndroFARsh** [comment]
@gpeal yes i have large args and several viewModel, also have 3-7 fragments in backstack. This leads to ttl.

Unfortunately this is a legacy code, and we haven't bandwidth to refactoring it right now.

Anyway i think, it would be great if library wouldn't duplicate args if we will have multiple viewModels. Or it would be good to have  flag to ignore args during save process

**Comment by gpeal** [comment]
@AndroFARsh I'd have to re-familiarize myself with that part of the code to know if/what a good solution would be. It probably won't be a high priority for us right now but if you want to take a look and potentially put up a PR, that could be helpful.
